lte/cloud/go/tools/migrations/m011_subscriber_config/main.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+
+	"magma/orc8r/cloud/go/sqorc"
+	"magma/orc8r/cloud/go/tools/migrations"
+
+	"github.com/Masterminds/squirrel"
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
+)
+
+const (
+	tableName = "cfg_entities"
+	pkCol     = "pk"
+	typeCol   = "type"
+	configCol = "config"
+
+	subscriberType = "subscriber"
+)
+
+type SubscriberConfig struct {
+	Lte json.RawMessage `json:"lte"`
+}
+
+// This migration updates the config of all subscriber entities to the
+// SubscriberConfig struct instead of the old LteSubscription struct.
+func main() {
+	dbDriver := migrations.GetEnvWithDefault("SQL_DRIVER", "postgres")
+	dbSource := migrations.GetEnvWithDefault("DATABASE_SOURCE", "dbname=magma_dev user=magma_dev password=magma_dev host=postgres sslmode=disable")
+	db, err := sqorc.Open(dbDriver, dbSource)
+	if err != nil {
+		glog.Fatal(errors.Wrap(err, "could not open db connection"))
+	}
+
+	_, err = migrations.ExecInTx(db, &sql.TxOptions{Isolation: sql.LevelSerializable}, nil, doMigration)
```

Comment:
I like this pattern of having just a single `doMigration` in main's ExecInTx -- easier to follow the logic